src/openlifu/db/database.py

```diff
@@ -456,26 +469,18 @@ def load_session_info(self, subject_id, session_id):
             raise FileNotFoundError(f"Session file not found for ID: {session_id}")
 
 
-    def load_solution(self, session, protocol_id, target_id):
-        solution_filename_json = self.get_solution_filename(session.subject_id, session.id, protocol_id, target_id, ext="json")
-        solution_filename_mat = self.get_solution_filename(session.subject_id, session.id, protocol_id, target_id, ext="mat")
+    def load_solution(self, session:Session, solution_id:str) -> Solution:
```

Comment:
In the matlab code, the function is defined as:
```matlab
[solution, output] = load_solution(self, session, plan_id, target_id) 
```
Currently It is missing the `output`

Reply:
Maybe you can check me on this but I think the `output` is the the `Solution.simulation_output` xarray here. In the  python library it is an attribute of `Solution` rather than a separate object. See [my answer below](https://github.com/OpenwaterHealth/OpenLIFU-python/pull/116#discussion_r1771330189) about what is in `Solution.simulation_output`

Reply:
Indeed I just checked, it will also contain the `analysis` output.